enterprise_access/apps/subsidy_access_policy/tests/mixins.py

```diff
@@ -44,17 +44,16 @@ def setUp(self):
 
         self.mock_lms_api_client = lms_api_client_patcher.start()
 
-        includes_user_patcher = patch.object(
-            SubsidyAccessPolicy, 'includes_user'
+        enterprise_user_record_patcher = patch.object(
+            SubsidyAccessPolicy, 'enterprise_user_record'
         )
 
-        self.mock_includes_user = includes_user_patcher.start()
-        self.mock_includes_user.return_value = True
+        self.mock_enterprise_user_record = enterprise_user_record_patcher.start()
```

Comment:
nit: it'd be good to set a reasonable default return value, like `self.mock_enterprise_user_record.return_value = {}`